src/components/Toggle/README.md

```diff
@@ -0,0 +1,142 @@
+---
+name: Toggle
+category: Forms
+platforms:
+  - web
+keywords:
+  - toggle
+  - switch
+  - selection
+  - choices
+  - options
+  - pick
+  - single selection form
+  - choice form
+  - option button
+  - enable
+  - disable
+---
+
+# Toggle
+
+Use to present a single choice to enable or disable a setting.
+
+---
+
+## Best practices
+
+Toggle should:
+
+- Always be used with an associated label component.
+- Be used to switch a single setting on and off.
+
+---
+
+## Content guidelines
+
+### Toggle labels
+
+Toggle labels should:
+
+- Start with a capital letter
+
+<!-- usagelist -->
+
+#### Do
+
+- Off
+
+#### Don’t
+
+- off
+
+<!-- end -->
+
+- Not end in punctuation if it’s a single sentence, word, or a fragment
+
+<!-- usagelist -->
+
+#### Do
+
+- Off
+
+#### Don’t
+
+- Off;
+
+<!-- end -->
+
+---
+
+## Examples
+
+### Default toggle
+
+Use toggle to let merchants enable or disable a setting.
+
+```jsx
+class RadioButtonExample extends React.Component {
+  state = {
+    checked: false,
+  };
+
+  toggleToggle = (value) => {
+    this.setState({checked: value});
+  };
+
+  render() {
+    const {value} = this.state;
+    return (
+      <Stack vertical>
+        <Toggle
+          label="Accept email marketing"
+          labelHidden
+          checked={this.state.checked}
+          onChange={this.toggleToggle}
+        />
```

Comment:
This example feels a bit too close to the example we're using for a Checkbox. What is the differentiator here? Shouldn't this be more like a settings toggle in the admin?

Reply:
@danrosenthal Do you mean that the state should not be called `checked`? Perhaps something like `enabled`?

Reply:
It's less to do with the state, and more to do with the content of the example. What I'm looking for is an example that shows me why I would want to use a `Toggle` vs. a `Checkbox`. Does that make sense?

Reply:
Yes!